native/src/main/java/bblfsh/bash/ASTNodeSerializer.java

```diff
@@ -35,6 +35,9 @@ public void serialize(ASTNode root, JsonGenerator jG, SerializerProvider provide
         final int length = root.getTextLength();
         jG.writeNumberField("textLength", length);
 
+        final String text = root.getText();
+        jG.writeStringField("text", text);
```

Comment:
Yes, this PR and #11 has a commit in common, which is why you see the changes twice.

I always get confused whenever this happens too and that's why I started this PR's text explaining that it is related with #11.

I would like to stop doing this kind of PR mixing, but I don't know how to do it without delaying PRs until their dependencies had been merged.  The rest of the team seems to be ok with this kind of workflow anyway.